src/components/products/ProductImage.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { MouseEvent, useCallback, useState } from 'react';
+
+type ProductImageProps = {
+  images: Image[];
+};
+
+type Image = {
+  imageSrc: string;
+  imageAlt: string;
+};
+
+export function ProductImage({ images }: ProductImageProps) {
```

Comment:
I think this defers from the image slider component made in figma, here are some indifferences or missing things:
- the selectable images are displayed vertically rather than horizontally
- the selectable images are displayed on the left of the default image rather then the bottom

Reply:
Well aware.

This was different implementation to see what works best.

So, what does? This, or the initial design?

Others can drop their suggestion here. @musllim @emilenfc @ISHIMWE-Aime

Reply:
From my point of view, I have seen that your design looks great.

Reply:
I can see that look great too